tfhe/src/shortint/server_key/mod.rs

```diff
@@ -367,7 +367,7 @@ impl ServerKey {
     /// let ct = cks.encrypt(msg);
     ///
     /// // Generate the lookup table for the function f: x -> x^2 mod 2^2
-    /// let f = |x| x ^ 2 % 4;
+    /// let f = |x| x ^ 2;
```

Comment:
this is most likely a wrong usage of ^, this should be a power instead what's the status on https://github.com/zama-ai/tfhe-rs/pull/638 @tmontaigu ?

Reply:
Its ready to be reviewed